testharnessreport.js

```diff
@@ -389,13 +389,20 @@ function dump_test_results(tests, status) {
     var test_results = tests.map(function(x) {
         return {name:x.name, status:x.status, message:x.message, stack:x.stack}
     });
-    data = {test:window.location.href,
+    var data = {test:window.location.href,
             tests:test_results,
```

Comment:
Please reindent the rest of the object.

Also, I would appreciate it if you rebased your PR, redistributing the changes into two commits: one to fix the global variables/whitespace, and one for the actual changes to fix #157.